tests/unit_tests/tests_launch/test_launch.py

```diff
@@ -1157,6 +1157,7 @@ def test_run_in_launch_context_with_malformed_env_vars(
         assert "Malformed WANDB_ARTIFACTS, using original artifacts" in err
 
 
+@pytest.mark.timeout(240)
```

Comment:
wow... this test takes a long time... maybe we should move to nightly?

Reply:
all i know is that it takes more than 60 seconds occasionally-- i just chose 240 be safe and that is what is done for other tests.

but yeah, we should move things around to nightly as needed.. but note that we only have yea in nightly right now, not pytest

Reply:
yeah... we probably don't want tests so long gating PRs...